Quotient/connection.cpp

```diff
@@ -1882,6 +1911,20 @@ bool Connection::isKnownE2eeCapableDevice(const QString& userId, const QString&
     return query.next();
 }
 
+bool Connection::hasConflictingDeviceIdsAndCrossSigningKeys(const QString& userId)
+{
+    if (d->encryptionData) {
+        return d->encryptionData->hasConflictingDeviceIdsAndCrossSigningKeys(userId);
+    }
+    return true;
+}
+
+void Connection::reloadDevices()
+{
+    if (d->encryptionData) {
+        reloadDevices();
```

Comment:
I seem to get an infinite loop here sometimes, and I can't really tell what it's supposed to be doing. Seems to be a rebasing error?

Reply:
yeah this should probably be d->encryptionData->reloadDevices()

Reply:
Can confirm, changing it to this prevents an infinite loop. Easiest way to tell is when you complete the verification on the other end before in NeoChat, because NeoChat will freeze :laughing:

Reply:
```suggestion
        d->encryptionData->reloadDevices();
```